server/install_dependencies_and_build.sh

```diff
@@ -36,7 +36,7 @@ apt update -q=2 \
     && . /etc/os-release \
     && echo "deb [signed-by=/usr/share/keyrings/kitware-archive-keyring.gpg] https://apt.kitware.com/ubuntu/ $UBUNTU_CODENAME main" | tee /etc/apt/sources.list.d/kitware.list >/dev/null \
     && apt-get update -q=2 \
-    && apt-get install -y --no-install-recommends cmake=3.27.7* cmake-data=3.27.7* \
+    && apt-get install -y --no-install-recommends cmake=3.28.* cmake-data=3.28.* \
```

Comment:
was this missed on original PR?
we should add patch number here to make it consistent with others

Reply:
Come out with suggestion

Reply:
version should be `cmake=3.28.3*`, similar to other repos - https://github.com/triton-inference-server/server/pull/7758/files
I can push the change, if you prefer that.

Reply:
Let me show you how is suggestion looks: 
```suggestion
    && apt-get install -y --no-install-recommends cmake=3.28.3* cmake-data=3.28.3* \
```

Reply:
looks good, thanks!